File: tests/machine/x/php/go_auto.php
<?php
$testpkg = [ 'Add' => function($a, $b) { return $a + $b; }, 'Pi' => 3.14, 'Answer' => 42 ];
var_dump($testpkg['Add'](2, 3));
var_dump($testpkg['Pi']);
var_dump($testpkg['Answer']);
?>

File: tests/machine/x/scala/go_auto.scala
object go_auto {
  object testpkg {
    def Add(a: Int, b: Int): Int = a + b
    val Pi: Double = 3.14
    val Answer: Int = 42
  }

  def main(args: Array[String]): Unit = {
    println((testpkg.Add(2, 3)))
    println((testpkg.Pi))
    println((testpkg.Answer))
  }
}
